ReleaseNotes.txt

```diff
@@ -90,4 +90,5 @@ Version 3.1.0
 - MW-537 Send image quality scores and model version to localize endpoint
 - MW-538 Add freeform tags to SDK API and send them to server
 - MW-547 End Ride button hidden for a while when opening SDK
-- MW-557 TooLittle frame count increases nearly 1to1 with every frame
\ No newline at end of file
+- MW-557 TooLittle frame count increases nearly 1to1 with every frame
```

Comment:
Could also say: TooLittle frame count and TooBlurry frame counts no longer increase when ARSession frame is null